commonui/src/test/java/com/hyperwallet/android/ui/common/util/TransferMethodUtilsTest.java

```diff
@@ -74,15 +75,15 @@ public void testGetTransferMethodName_returnsCorrectNameForType() throws JSONExc
         HyperwalletTransferMethod hyperwalletTransferMethod = new HyperwalletTransferMethod(htmJsonObject);
         when(mContext.getString(R.string.bank_card)).thenReturn("My card");
         String transferMethodName = TransferMethodUtils.getTransferMethodName(mContext, hyperwalletTransferMethod);
-        assertThat(transferMethodName, is("My card"));
+        MatcherAssert.assertThat(transferMethodName, Matchers.is("My card"));
```

Comment:
please import the whole static class for `MatcherAssert` and `Matchers` please

Reply:
removed